ngx-sub-form doesn't build with Angular's Ivy compiler

So, Angular 9 just dropped today, and the Ivy compiler is now default. But if I try to build one of my Angular pages that uses ngx-sub-form while using Ivy, I get the following error:
```
ERROR in CommissionedArt/src/app/Components/input-form.html:1:26 - error TS2740: Type 'TypedFormGroup<InputForm>' is missing the following properties from type 'FormGroup': _parent, _asyncValidationSubscription, _updateAncestors, _setInitialStatus, and 4 more.
```
I don't get this if I disable Ivy and use the old View Engine compiler instead.

Oh, I probably should also mention that I got this error alongside that one:
```
CommissionedArt/src/app/app.html:1:13 - error TS2322: Type 'InputForm | null' is not assignable to type 'InputForm'.
  Type 'null' is not assignable to type 'InputForm'.
```
That one I could solve by just making my dataInput type have `| null` on it, but it would be nice to not require that too.

--- a/projects/ngx-sub-form/src/lib/ngx-sub-form.types.spec.ts
+++ b/projects/ngx-sub-form/src/lib/ngx-sub-form.types.spec.ts
@@ -1,0 +1,14 @@
+import { FormControl, FormGroup } from '@angular/forms';
+import { TypedFormGroup } from 'ngx-sub-form';
+
+describe(`TypedFormGroup`, () => {
+  it(`should be assignable to the base @angular/forms FormGroup`, () => {
+    let formGroup: FormGroup;
+
+    const typedFormGroup = new FormGroup({ foo: new FormControl() }) as TypedFormGroup<{ foo: true }>;
+
+    formGroup = typedFormGroup;
+
+    expect(true).toBe(true); // this is a type-only test, if the type breaks the test will not compile
+  });
+});

--- a/projects/ngx-sub-form/src/lib/ngx-sub-form.types.ts
+++ b/projects/ngx-sub-form/src/lib/ngx-sub-form.types.ts
@@ -8,15 +8,14 @@ export interface OnFormUpdate<FormInterface> {
   onFormUpdate?: (formUpdate: FormUpdate<FormInterface>) => void;
 }
 
-type Omit<T, K> = Pick<T, Exclude<keyof T, K>>;
 type Nullable<T> = T | null;
 
 export type NullableObject<T> = { [P in keyof T]: Nullable<T[P]> };
 
-export type TypedFormGroup<FormInterface> = Omit<FormGroup, 'controls' | 'value'> & {
+export interface TypedFormGroup<FormInterface> extends FormGroup {
   controls: Controls<FormInterface>;
   value: FormInterface;
-};
+}
 
 export type TypedValidatorFn<T> = (formGroup: TypedFormGroup<T>) => ValidationErrors | null;
 

